libsplinter/src/circuit/service.rs

```diff
@@ -70,7 +70,7 @@ impl Service {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
 pub struct SplinterNode {
-    #[serde(skip)]
+    #[serde(default)]
```

Comment:
Typos in commit message: s/no/now/ and s/backwords/backwards/

Reply:
fixed